ksql-engine/src/main/java/io/confluent/ksql/util/KafkaConsumerGroupClientImpl.java

```diff
@@ -49,9 +57,22 @@ public KafkaConsumerGroupClientImpl(KsqlConfig ksqlConfig) {
 
   @Override
   public List<String> listGroups() {
+    File tmpConfigFile;
```

Comment:
Consider moving all the parts needed to create the admin client args into a `getAdminClientArgs()`